garage/tf/misc/tensor_utils.py

```diff
@@ -10,6 +13,50 @@ def run(*input_vals):
     return run
 
 
+def flatten_batch(t, name=None):
+    with tf.name_scope(name, "flatten_batch", [t]):
+        shape = [-1] + list(t.shape[2:])
+        return tf.reshape(t, shape)
+
+
+def flatten_batch_dict(d, name=None):
```

Comment:
Can you verify that these don't already exist elsewhere?

Reply:
I didn't find it elsewhere but I'm not sure. I asked Jonathon for review because he refactored some utils in garage and garage/tf.

Reply:
I don't think we have this. Maybe you can consder placing these in a separate utils function for garage/tf (the only one we have right now is located in garage/envs/util)

Reply:
I think it might be a feature of spaces?

Overall we need a strong convention for the shape of sampled data, and then it will be easy to produce flattened views in any NN framework.

The shape used here, which is fairly general, is (trajectory, rollout, data_1, data_2, ..., data_n) where the shape of a single data point is (data_1, ..., data_n).

We can file an issue make this uniform across the codebase later.